test/Twilio.Test/ClusterTest.cs

```diff
@@ -0,0 +1,126 @@
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using Twilio;
+using Twilio.Rest.Api.V2010.Account;
+using Twilio.Rest.Chat.V2;
+using Twilio.Rest.Chat.V2.Service;
+using Twilio.Rest.Events.V1;
+using System.Linq;
+namespace Twilio.Tests 
+{
+    [TestFixture]
+    class ClusterTest 
+    {
+        string   accountSid;
+        string  secret;
+        private string  apiKey;
+        string  toNumber;
+        string  fromNumber;
+        [SetUp]
+        [Category("ClusterTest")]
+        public void SetUp()
+        {
+            accountSid = Environment.GetEnvironmentVariable("TWILIO_ACCOUNT_SID");
+            secret = Environment.GetEnvironmentVariable("TWILIO_API_SECRET");
+            apiKey = Environment.GetEnvironmentVariable("TWILIO_API_KEY");
+            toNumber = Environment.GetEnvironmentVariable("TWILIO_TO_NUMBER");
+            fromNumber = Environment.GetEnvironmentVariable("TWILIO_FROM_NUMBER");
+            TwilioClient.Init(username:apiKey,password:secret,accountSid:accountSid);
+        }
+        
+
+        [Test]
+        [Category("ClusterTest")]
+        public void TestSendingAText()
+        {
+             var message = MessageResource.Create(
+                from: new Twilio.Types.PhoneNumber(fromNumber),
+                body: "Where's Wallace?",
+                to: new Twilio.Types.PhoneNumber(toNumber)
+            );
+            Assert.IsNotNull(message);
+            Assert.True(message.Body.Contains("Where's Wallace?"));
+            Assert.AreEqual(fromNumber,message.From.ToString());
+            Assert.AreEqual(toNumber,message.To.ToString());
```

Comment:
We need to delete whatever resource we are creating as a part of these cluster test at the end as we are hitting prod.